tests/core/pyspec/eth2spec/test/phase0/fork_choice/test_get_head.py

```diff
@@ -0,0 +1,288 @@
+from eth_utils import encode_hex
+
+from eth2spec.test.context import MINIMAL, with_all_phases, with_configs, spec_state_test
+from eth2spec.test.helpers.attestations import get_valid_attestation, next_epoch_with_attestations
+from eth2spec.test.helpers.block import build_empty_block_for_next_slot
+from eth2spec.test.helpers.fork_choice import (
+    add_attestation_to_store,
+    add_block_to_store, get_anchor_root,
+    get_genesis_forkchoice_store_and_block,
+    get_attestation_file_name,
+    get_block_file_name,
+)
+from eth2spec.test.helpers.state import (
+    next_epoch,
+    state_transition_and_sign_block,
+)
+
+
+@with_all_phases
+@with_configs([MINIMAL])
+@spec_state_test
+def test_genesis(spec, state):
```

Comment:
For Altair, this produces an Altair state at genesis, and Altair blocks right? The test is good (we may start Altair testnets without forking into Altair) but maybe this is a good place to use that `description` field to explain why it's used post-phase0.